packages/vue-query/src/__tests__/useQueries.test.ts

```diff
@@ -159,6 +159,8 @@ describe('useQueries', () => {
   })
 
   test('should stop listening to changes on onScopeDispose', async () => {
+    const consoleMock = jest.spyOn(console, 'error')
+    consoleMock.mockImplementation(() => undefined)
```

Comment:
why do we need a mock here? it doesn't look like the test would log anything on purpose?